### ISSUE TITLE
keras contrib is deprecated, 

### ISSUE THREAD

**Issue by EngrStudent** [eriklindernoren/Keras-GAN#206]
the "keras_contrib.layers.normalization.instancenormalization" has been deprecated, and the content moved to tensorflow-addons.  
https://github.com/keras-team/keras-contrib

Also, it requires pip to install instead of just conda, which makes it a little less easy to use.

The disco-gan uses keras contrib.  You might update to more recent "textbook" tensorflow.
